batch/sql/estimated-current.sql

```diff
@@ -1562,6 +1688,22 @@ BEGIN
       WHERE id = in_batch_id;
     END IF;
 
+    # update the rest of the non-root job groups if they exist
+    # necessary for backwards compatibility
+    UPDATE batches_n_jobs_in_complete_states
+    INNER JOIN (
+      SELECT batch_id, parent_id
+      FROM job_group_parents
+      WHERE batch_id = in_batch_id AND job_group_id = cur_job_group_id AND job_group_id != 0
+      ORDER BY job_group_id ASC
+    ) AS t ON batches_n_jobs_in_complete_states.id = t.batch_id AND batches_n_jobs_in_complete_states.job_group_id = t.parent_id
+    SET n_completed = n_completed + 1,
+        n_cancelled = n_cancelled + (new_state = 'Cancelled'),
+        n_failed = n_failed + (new_state = 'Error' OR new_state = 'Failed'),
+        n_succeeded = n_succeeded + (new_state != 'Cancelled' AND new_state != 'Error' AND new_state != 'Failed');
```

Comment:
Just to be clear: this has absolutely no effect in this PR because it is not possible to create job groups (other than the implicitly created root one)?

Reply:
I believe that is correct. I believe I can move this into a different PR that will need to be done after the job groups tables are fully populated that allow for job_group_id != 0 as well as having explicit primary keys with job_group_id in the tables for `aggregated_batch_resources`, `batches_cancelled` etc.